docs/developers/developing_iree/profiling_with_tracy.md

```diff
@@ -48,6 +48,17 @@ make -C third_party/tracy/profiler/build/unix -j12 release
 
 TODO write this (Kojo?)
 
+
+## Building the Tracy Command Line Capture (the alternative "server")
+
+Build and run the command line capture when you don't have a UI or can't SSH in to
```

Comment:
Does the version we build not work? I tried to do it right so that it should be binary compatible with just about every system.

Reply:
Or you can set this and it will spit out a headless server to your build dir: https://github.com/google/iree/blob/47b334decbd1ee93fad0b414ffdf11dd3466a005/CMakeLists.txt#L43

Reply:
woot. this is all I needed. I was trying to follow the guidance on discord to get it to work. But this should be the easiest way. Let me test it.

Reply:
works.